pkg/kubelet/container_bridge.go

```diff
@@ -109,6 +84,49 @@ func ensureCbr0(wantCIDR *net.IPNet, promiscuous, babysitDaemons bool) error {
 	return nil
 }
 
+func createCBR0(br *netlink.Bridge, wantCIDR *net.IPNet, babysitDaemons bool) error {
+	addr := &netlink.Addr{
+		IPNet: wantCIDR,
+		Label: "",
+	}
+
+	if err := netlink.LinkAdd(br); err != nil && err != syscall.EEXIST {
```

Comment:
Shouldn't the bridge not exist at this point, unless something external is racing with kubelet?  Also, if the error is EEXIST couldn't the AddrAdd() call fail with EEXIST too?  It's not clear to me why we ignore EEXIST for the bridge itself but not for the address.